
/tof_sensor:
  ros__parameters:
    flip_horizontal: false
    flip_vertical: false
    stream_type: distance_amplitude
